R/rep.R

```diff
@@ -83,14 +85,22 @@ NULL
 
 #' @rdname vec-rep
 #' @export
-vec_rep <- function(x, times) {
-  .Call(ffi_vec_rep, x, times)
+vec_rep <- function(x,
+                    times,
```

Comment:
Strongly recommend adding empty `...` in both of these functions to require named optional args

Reply:
oh yeah